api/app/api.py

```diff
@@ -39,7 +43,7 @@ async def read_root() -> dict:
 
 @app.get("/get_products")
 async def read_db_products() -> dict:
-    with open(f'{DIR_PATH}\\app\\database.json') as json_file:
+    with open(f'{DIR_PATH}/app/database.json') as json_file:

```

Comment:
For OS independent paths use the r"mypath/subdirectory" syntax and os pathlib module

Reply:
done, sir